addons/mail/static/src/new/thread/attachment_list.scss

```diff
@@ -0,0 +1,46 @@
+.o-mail-attachment-list {
+    max-width: var(--Chatter-max-width, none);
+}
```

Comment:
I don't see any change from this LOC. I remove it, as it refers to an unused variable in new code.